cli/azd/pkg/devcenter/provision_provider.go

```diff
@@ -207,7 +207,7 @@ func (p *ProvisionProvider) Deploy(ctx context.Context) (*provisioning.DeployRes
 
 	go p.pollForEnvironment(pollingContext, envName)
 
-	_, err = poller.PollUntilDone(ctx, nil)
+	_, _ = poller.PollUntilDone(ctx, nil)
```

Comment:
Why do we ignore the error here? If we mean to do this, does it mean the code on L211 is now incorrect?

Reply:
This part will be rolled back - there is a bug on the ADE side that is always shows bicep in failed state.